pmd-java/src/main/resources/rulesets/java/quickstart.xml

```diff
@@ -0,0 +1,330 @@
+
+    <rule ref="category/java/errorprone.xml/AssignmentInOperand">
+        <properties>
+            <property name="allowWhile" value="true"/>
+        </properties>
+    </rule>
+    <rule ref="category/java/errorprone.xml/AssignmentToNonFinalStatic"/>
+    <rule ref="category/java/errorprone.xml/AvoidAccessibilityAlteration"/>
+    <!-- <rule ref="category/java/errorprone.xml/AvoidAssertAsIdentifier" /> -->
+    <rule ref="category/java/errorprone.xml/AvoidBranchingStatementAsLastInLoop"/>
+    <!-- <rule ref="category/java/errorprone.xml/AvoidCallingFinalize" /> -->
+    <!-- <rule ref="category/java/errorprone.xml/AvoidCatchingNPE" /> -->
+    <rule ref="category/java/errorprone.xml/AvoidCatchingThrowable"/>
+    <rule ref="category/java/errorprone.xml/AvoidDecimalLiteralsInBigDecimalConstructor"/>
+    <!-- <rule ref="category/java/errorprone.xml/AvoidDuplicateLiterals" /> -->
+    <!-- <rule ref="category/java/errorprone.xml/AvoidEnumAsIdentifier" /> -->
+    <!-- <rule ref="category/java/errorprone.xml/AvoidFieldNameMatchingMethodName" /> -->
+    <!-- <rule ref="category/java/errorprone.xml/AvoidFieldNameMatchingTypeName" /> -->
+    <rule ref="category/java/errorprone.xml/AvoidInstanceofChecksInCatchClause"/>
+    <!-- <rule ref="category/java/errorprone.xml/AvoidLiteralsInIfCondition" /> -->
+    <!-- <rule ref="category/java/errorprone.xml/AvoidLosingExceptionInformation" /> -->
+    <rule ref="category/java/errorprone.xml/AvoidMultipleUnaryOperators"/>
+    <rule ref="category/java/errorprone.xml/AvoidUsingOctalValues"/>
+    <rule ref="category/java/errorprone.xml/BadComparison"/>
+    <!-- <rule ref="category/java/errorprone.xml/BeanMembersShouldSerialize" /> -->
+    <rule ref="category/java/errorprone.xml/BrokenNullCheck"/>
+    <!-- <rule ref="category/java/errorprone.xml/CallSuperFirst" /> -->
+    <!-- <rule ref="category/java/errorprone.xml/CallSuperLast" /> -->
+    <rule ref="category/java/errorprone.xml/CheckSkipResult"/>
+    <rule ref="category/java/errorprone.xml/ClassCastExceptionWithToArray"/>
+    <rule ref="category/java/errorprone.xml/CloneMethodMustBePublic"/>
+    <rule ref="category/java/errorprone.xml/CloneMethodMustImplementCloneable"/>
+    <rule ref="category/java/errorprone.xml/CloneMethodReturnTypeMustMatchClassName"/>
+    <rule ref="category/java/errorprone.xml/CloneThrowsCloneNotSupportedException"/>
+    <rule ref="category/java/errorprone.xml/CloseResource"/>
+    <rule ref="category/java/errorprone.xml/CompareObjectsWithEquals"/>
+    <!-- <rule ref="category/java/errorprone.xml/ConstructorCallsOverridableMethod" /> -->
+    <!-- <rule ref="category/java/errorprone.xml/DataflowAnomalyAnalysis" /> -->
+    <rule ref="category/java/errorprone.xml/DoNotCallGarbageCollectionExplicitly"/>
+    <!-- <rule ref="category/java/errorprone.xml/DoNotCallSystemExit" /> -->
+
+    <rule ref="category/java/errorprone.xml/DoNotExtendJavaLangThrowable"/>
```

Comment:
DoNotExtendJavaLangError is not included. I see no reason for having one but not the other